calico-enterprise/visibility/elastic/archive-storage.mdx

```diff
@@ -22,6 +22,14 @@ This how-to guide uses the following {{prodname}} features:
 
 - **LogCollector** resource
 
+## Before you begin
+
+**Supported logs for export**
+
+- Syslog - flow, dns, idsevents, audit
+- Amazon S3 - l7, flow, dns, runtime, compliance, audit
```

Comment:
[In the ticket](https://tigera.atlassian.net/browse/DOCS-1418?focusedCommentId=64261) it's mentioned that we will not be adding runtime for enterprise, did that change ?

Reply:
Sorry, I forgot to commit my changes...you should see "runtime" only the CC version of the docs.